dev-requirements.txt

```diff
@@ -8,4 +8,5 @@ git+git://github.com/PyCQA/pylint.git#egg=pylint
 pytest
 taskcat
 yq
-troposphere
\ No newline at end of file
+troposphere
```

Comment:
sorry this new line change happened again i will figure out how to fix after this airflow update